code/game/machinery/lightswitch.dm

```diff
@@ -182,3 +182,7 @@
 		light_connect = !light_connect
 	if("toggle_logic" in href_list)
 		logic_connect = !logic_connect
+
+/obj/machinery/light_switch/off/Initialize()
+	..()
+	attack_hand()
```

Comment:
```suggestion
/obj/machinery/light_switch/off/Initialize()
	. = ..()
	attack_hand()
```

This needs a return value. I would also rather it be a preset with the switch off, if thats doable by setting the area or something else.

(Basically if you have multiple of these switches place they could all toggle and affect eachother)

Reply:
Since the place starts depowered, I figured it makes no sense to have an off-switch at all... I replaced this with a regular light switch.